src/globus_sdk/services/flows/client.py

```diff
@@ -668,6 +668,38 @@ def update_run(
 
         return self.put(f"/runs/{run_id}", data=data)
 
+    def delete_run(self, run_id: UUIDLike) -> GlobusHTTPResponse:
+        """
+        Delete a run.
+
+        :param run_id: The ID of the run to update
+        :type run_id: str or UUID
+
+
+        .. tab-set::
+
+            .. tab-item:: Example Usage
+
+                .. code-block:: python
+
+                    from globus_sdk import FlowsClient
+
+                    flows = FlowsClient(...)
+                    flows.delete_run("581753c7-45da-43d3-ad73-246b46e7cb6b")
+
+            .. tab-item:: Example Response Data
+
+                .. expandtestfixture:: flows.delete_run
+
+            .. tab-item:: API Info
+
+                .. extdoclink:: Delete Run
+                    :service: flows
+                    :ref: Runs/paths/~1runs~1{run_id}~1release/post
```

Comment:
Caught this after approving, looks like this is not rendering right in the docs. When I click the preview it takes me to: https://globusonline.github.io/flows/#tag/Runs/paths/~1runs~1%7Brun_id%7D~1release/post

but the actual location is: https://globusonline.github.io/flows/#tag/Runs/paths/~1runs~1{action_id}~1release/post

I know that the `run_id` change will work out as soon as the Flows OpenAPI docs are redeployed, but I think the urlencoding of the curly braces might still be broken (tested this manually in the browser by replacing with `action_id` to go to the current location but leaving the urlencoded entities and it didn't seem to resolve correctly for me).